docs/managed-datahub/observe/assertions.md

```diff
@@ -0,0 +1,53 @@
+# Assertions
+
+:::note Contract Monitoring Support
+Currently we support Snowflake, Databricks, Redshift, and BigQuery for out-of-the-box contract monitoring as part of Acryl Observe.
+:::
+
+
+## What is an Assertion
+
+An assertion is **a data quality test that finds data that violate one or more specified rules.** These serve as the backbone of [Data Contracts](/docs/managed-datahub/observe/data-contract.md) – this is how we verify the contract is met.
+
+## How to Create and Run Assertions
+
+Data quality tests (a.k.a. assertions) can be ingested from a 3rd party tool or run by Acryl.
```

Comment:
yes, but they can also be created through the UI when using Acryl Cloud. also the 3rd party tools are more complex, and so that section should be second